angular-workspace/projects/ni/nimble-angular/test.ts

```diff
@@ -30,7 +32,6 @@ context.keys().map(context);
 
 // Workaround to make console.error calls fail tests:
 // https://github.com/angular/angular/issues/36430#issuecomment-874772398
-// Might be removable after upgrading to Angular 14:
-// https://github.com/angular/angular/issues/36430#issuecomment-1117801535
+// This can be useful to have even with errorOnUnknownElements and errorOnUnknownProperties set to true.
```

Comment:
Can simplify the comments to something like:
`// Elevate console errors to test failures`